server/src/api/login.go

```diff
@@ -28,21 +28,21 @@ func (s *Server) signInAnonymously(w http.ResponseWriter, r *http.Request) {
 
 	var body AnonymousSignUpRequest
 	if err := render.Decode(r, &body); err != nil {
-		w.WriteHeader(http.StatusBadRequest)
+		common.Throw(w, r, common.BadRequestError(err))
 		return
 	}
 
 	user, err := s.users.LoginAnonymous(r.Context(), body.Name)
 	if err != nil {
 		log.Errorw("could not create user", "req", body, "err", err)
-		common.Throw(w, r, common.InternalServerError)
+		common.Throw(w, r, err)
 		return
 	}
 
 	tokenString, err := s.auth.Sign(map[string]interface{}{"id": user.ID})
 	if err != nil {
 		log.Errorw("unable to generate token string", "err", err)
-		common.Throw(w, r, common.InternalServerError)
+		common.Throw(w, r, err)
```

Comment:
why aren't you throwing interval server error explicitly anymore?

Reply:
I think the plan was to return a more meaningful error.
But common.Throw() also just renders an InternalServerError when not specifying an ApiError.
Either way nothing really changed and my changes were meaningless :D
I'll change it back :)

Reply:
no it's fine then 👍  
Or alternatively we could add more custom errors to get more variety than 401, 404 and 500?